app/markets/ActivityTable.tsx

```diff
@@ -177,9 +185,141 @@ export const ActivityTable = ({ id }: { id: string }) => {
   );
 };
 
+interface TradeRowProps {
+  activity: MergedTradeTransaction;
+  isAIAgent: boolean;
+}
+
+const TradeRow = ({ activity, isAIAgent }: TradeRowProps) => {
+  const creatorAddress = activity?.creator?.id ?? 'unknown';
+  const outcomes = activity.fpmm.outcomes;
+  const outcomeIndex = activity.outcomeIndex;
+
+  if (!outcomes || !outcomeIndex) return null;
+
+  const hasOutcomes = outcomes && outcomeIndex;
+
+  const outcome = hasOutcomes
+    ? new Outcome(
+        outcomeIndex,
+        outcomes[outcomeIndex] ?? 'Option ' + outcomeIndex,
+        activity.id
+      )
+    : null;
+
+  return (
+    <TableRow>
+      <TableCell className="flex items-center space-x-2 pl-4 text-text-high-em">
+        <a
+          href={getGnosisAddressExplorerLink(creatorAddress)}
+          className="hover:underline"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {shortenAddress(creatorAddress)}
+        </a>
+        {isAIAgent && <Image src="/ai.svg" alt="ai" width={18} height={18} />}
+      </TableCell>
+
+      <TableCell className="truncate text-text-high-em">
+        <div className="flex items-center space-x-2">
+          <p>
+            {activity.transactionType?.toLowerCase() === 'sell' && (
+              <span className="mr-0.5 text-md font-bold">-</span>
+            )}
+            {activity.transactionType?.toLowerCase() === 'buy' && (
+              <span className="mr-0.5 text-md font-bold">+</span>
+            )}
```

Comment:
Let use TransactionType here.

```suggestion
            {activity.transactionType?.toLowerCase() === TransactionType.Sell && (
              <span className="mr-0.5 text-md font-bold">-</span>
            )}
            {activity.transactionType?.toLowerCase() === TransactionType.Buy && (
              <span className="mr-0.5 text-md font-bold">+</span>
            )}
```